content/en/agent/faq/how-datadog-agent-determines-the-hostname.md

```diff
@@ -64,14 +46,76 @@ When pulling information on your AWS hosts from the [Datadog API][2], the follow
 
 {{< img src="agent/faq/host_aliases.png" alt="Host aliases" responsive="true" >}}
 
+## Agent versions
+
+{{< tabs >}}
+{{% tab "Agent v6" %}}
+
+There are differences in hostname resolution between Agent v5 and Agent v6.
+
+### Linux/MacOS
+
+When upgrading from Agent v5 to Agent v6, there might be a difference in the hostname reported by your Agent. To resolve the system hostname Agent v5 uses the `hostname -f` command while Agent v6 uses the Golang API `os.Hostname()`. On upgrades, the Agent hostname could change from a Fully-Qualified Domain Name (FQDN) to a short hostname, for example:
+
+`sub.domain.tld` --> `sub`
+
+**Note**: Starting from Agent v6.3, a configuration option called `hostname_fqdn` was introduced that allows Agent v6 to have the same behavior as Agent v5. This flag is disabled by default on version 6.3+. See the [example datadog.yaml][1] for enabling this option.
+
+#### Determine if you're affected
+
+Starting with v6.3.0, the Agent logs the warning below if you’re affected by the change:
+```text
+DEPRECATION NOTICE: The agent resolved your hostname as <HOSTNAME>. However in a future version, it will be resolved as <FQDN> by default. To enable the future behavior, please enable the `hostname_fqdn` flag in the configuration.
+```
+
+You are not affected if any of the following are true:
+
+* You are running the Agent in GCE.
+* You are setting the hostname in the [Agent's main configuration][2] file or through the `DD_HOSTNAME` environment variable.
+* You are running the Agent in a container with access to the Docker or Kubernetes API.
+* The hostname output of `cat /proc/sys/kernel/hostname` and `hostname -f` are the same.
+
+#### Recommended action
+
+If you're affected by this change, Datadog recommends taking the following action when you upgrade your Agent:
+
+* **Upgrading from Agent v5 to Agent v < 6.3**: Hardcode your hostname in the [Agent's main configuration][2] file.
+* **Upgrading from Agent v5 to Agent >= v6.3**: enable the `hostname_fqdn` option in the [Agent's main configuration][2] file. This ensures that you keep the same hostname.
+* **Upgrading from Agent v5 to Agent v6 (a future version which uses the fqdn by default)**: you don’t need to take any action.
+* If you want to ensure the current default behavior of Agent v6 is preserved when you upgrade the Agent in the future, set `hostname_fqdn` to `false`. That said, Datadog recommends you switch `hostname_fqdn` to `true` whenever possible.
+
+### Windows
+
+On Agent v5, the Windows Agent reported the unqualified hostname by default. To maintain backward compatibility, this behavior is preserved with Agent v6. The new flag `hostname_fqdn` remains disabled by default on Windows, and will remain disabled by default on future **v6** versions.
+
+The Windows Agent honors the configuration flag starting with v6.5. Setting `hostname_fqdn` to true results in the Windows Agent reporting the fully qualified hostname.
+
+#### Recommended action
```

Comment:
It seems that the H4 css layout is broken in tabs, instead of having the purple colors it seems to default to the H3 design